jet-fuel/test/controllers/api/vi/urls_controller_test.rb

```diff
@@ -0,0 +1,31 @@
+require "test_helper"
+
+class  Api::V1::UrlsControllerTest < ActionController::TestCase
+  test "#redirect" do
+    url = Url.create(full_url: "http://www.turing.io")
+    get :redirect, short_url: url.short_url, format: :json
+
+    full_url = response.body
+    assert_response :success
+    assert_equal full_url, url['full_url']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.